### PR TITLE
Fixes half covering glassware protecting eyes

### PR DESCRIPTION
**Pull request by Hoolny**
## About The Pull Request
Makes glasses that don't cover both eyes protect from dropper chemicals onto eyes
## Why It's Good For The Game
It's annoying to spray something on someones exposed eye for it to not transfer

MEDICS HAVE HAD IT GOOD FOR TO LONG resisting MY humanizing toxin with their dumb one eye covering health huds THE MED MOTHS SHALL BE HUMAN
## Changelog
:cl:
fix: fixes half-covering glassware protecting eyes from chemicals
/:cl:

### CODE REVIEW

**Comment by DaCoolBoss** [comment]
Does this affect screwdriver eye stabs and traitor eyestealers?

**Comment by SethLafuente** [comment]
> Does this affect screwdriver eye stabs and traitor eyestealers?

It essentially makes it so they no longer have the eye covering trait so yes it does affect those things as it should

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -666,12 +680,14 @@
 	desc = "Shows you know how to sew a lapel and center a back vent."
 	icon_state = "glasses_binoclard"
 	inhand_icon_state = null
+	flags_cover = GLASSESCOVERSEYES
```
Comment (Ghommie) [inline code/modules/clothing/glasses/_glasses.dm:683]: T'is a subtype of regular glasses, which already has the flag.
```suggestion
```

File: code/modules/clothing/glasses/hud.dm [hunk 2]
```diff
@@ -287,3 +295,4 @@
 	desc = "These sunglasses are special, and let you view potential criminals."
 	icon_state = "sun"
 	inhand_icon_state = "sunglasses"
+	flags_cover = GLASSESCOVERSEYES
```
Comment (Ghommie) [inline code/modules/clothing/glasses/hud.dm:298]: subtype of the spacecop glasses, which already has the flag
```suggestion
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/clothing/glasses/hud.dm (hunk 2 above)
Comment (SethLafuente) [reply code/modules/clothing/glasses/hud.dm:298]: knew at least a few where gona slip up

**Pull request merged**